utils.bal

```diff
@@ -0,0 +1,486 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/time;
+import ballerina/http;
+import ballerina/crypto;
+import ballerina/encoding;
+import ballerina/stringutils;
+import ballerina/lang.'string as str;
+import ballerina/lang.array as array;
```

Comment:
IINM, we don't need to explicitly import with an alias using `as` here, last word after `.` is treated as the alias.

Reply:
Applied this for all the imports